server/models/User.js

```diff
@@ -41,7 +42,7 @@ function defineModel() {
           },
           isBurnerEmail: function (val) {
             if (
-              this.changed('email') &&
+              this.emailChanged &&
```

Comment:
@znarf I'm curious, why add our own flag rather than use `this.changed('email')`?

Reply:
@Betree It was not working. And I could not find any documentation about it.

Reply:
Thank you for the link to the documentation. Not sure why I missed it.

I clearly remember it was not working and this is why I had to do this follow up. Maybe its because we tried to use it in a validator hook and that was an unsupported  edge case ?